README.md

```diff
@@ -1,2 +1,49 @@
-# Main-2022-09-22
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> For this project, I had to create "Hello world" repository and a feature branch that i cloned to my local machine, I then added index.HTML file and a Styles.css File, I set up linters for HTML&CSS and then created a .gitignore file to make sure certain files do not get published when i commit my changes from my local repo to my GitHub Master Repo, The goal here is to master all of the tools and best practices learned about in previous steps of The Program, which I will be using in all Microverse projects and most likely in my future job as well, so it is important to know the processes required!
+
+
+## Built With
+
+- HTML & CSS
+- Github Workflow Actions, HTML&CSS Linters
+- Virtual Studio Code, GitHub
+
+
+
+## Getting Started
+
+**This Project was to make sure i understand the basic tools and methods i will be using on a daily basis during the course of my microverse programm as well as future work life.**
+
+
+
+## Authors
+
+👤 **Jerome**
+
+- GitHub: [@githubhandle](https://github.com/187jjay187)
+- Twitter: [@twitterhandle](https://twitter.com/187jjay187)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/jerome-osman-137605a4)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Your link to the MIT license is broken. `Hint`, replace  `LICENCE` with `MIT.md`. Make sure you customize the `MIT.md` file with your personal  info `(where required)`. This would make your readme file more professional

Reply:
@frank1738 thank you for the review, I've added a description to my pull request and I've Fixed the broken link in my README.md file, please let me know if this is correct